src/libraries/System.Drawing.Common/Directory.Build.props

```diff
@@ -2,8 +2,7 @@
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
     <StrongNameKeyId>Open</StrongNameKeyId>
-    <IncludePlatformAttributes>true</IncludePlatformAttributes>
-    <UnsupportedOSPlatforms>browser</UnsupportedOSPlatforms>
+    <SupportedOSPlatforms>windows</SupportedOSPlatforms>
     <PackageDescription>Provides access to GDI+ graphics functionality.
```

Comment:
Consider adding detail to PackageDescription about lack of Unix support.

Reply:
Done, thanks.